bitnami/kong/README.md

```diff
@@ -0,0 +1,326 @@
+| `image.tag`                            | kong image tag                                                                                      | `{TAG_NAME}`                                            |
+| `image.pullPolicy`                     | kong image pull policy                                                                              | `IfNotPresent`                                          |
+| `image.pullSecrets`                    | Specify docker-registry secret names as an array                                                       | `[]` (does not add image pull secrets to deployed pods) |
+| `ingressControllerImage.registry`                       | kong ingress controller image registry                                                                                 | `docker.io`                                             |
+| `ingressControllerImage.repository`                     | kong ingress controller image name                                                                                     | `bitnami/kong`                                       |
+| `ingressControllerImage.tag`                            | kong ingress controller image tag                                                                                      | `{TAG_NAME}`                                            |
+| `ingressControllerImage.pullPolicy`                     | kong ingress controller image pull policy                                                                              | `IfNotPresent`                                          |
+| `ingressControllerImage.pullSecrets`                    | Specify docker-registry secret names as an array                                                       | `[]` (does not add image pull secrets to deployed pods) |
+| `migrationImage.registry`                       | kong migration job image registry                                                                                 | `docker.io`                                             |
+| `migrationImage.repository`                     | kong migration job image name                                                                                     | `bitnami/kong`                                       |
+| `migrationImage.tag`                            | kong migration job image tag                                                                                      | `{TAG_NAME}`                                            |
+| `migrationImage.pullPolicy`                     | kong migration job image pull policy                                                                              | `IfNotPresent`                                          |
+| `migrationImage.pullSecrets`                    | Specify docker-registry secret names as an array                                                       | `[]` (does not add image pull secrets to deployed pods) |
```

Comment:
I don't like this approach very much. You have a set of parameters to configure the Kong migration job, and another set to configure Kong Ingress Controller. Then, why not including the images under those section as we do in other charts? I mean, use: `migration.image.XXX` and `ingressController.image.YYY`

Reply:
Done